app/models/button_maker.rb

```diff
@@ -45,6 +45,8 @@ def set_item_properties
     @z30status = @item.xpath('z30/z30-item-status').text
     # Yes, this excludes `z30/` on purpose.
     @z30status_code = @item.xpath('z30-item-status-code').text
+    @year = @item.xpath('z13/z13-year').text
+    @volume = @item.xpath('z30/z30-description').text
```

Comment:
What's in this field when it isn't a journal? Will we be sad?

Reply:
I'm reasonably certain "it works" but uh... I'll get back to you.

Reply:
Yeah, it just adds a blank string if there is nothing there which seems fine enough.
https://mit-bento-staging-pr-297.herokuapp.com/record/cat00916a/mit.001739356